packages/components/src/DataList/DataList.stories.tsx

```diff
@@ -142,3 +143,110 @@ export const EmptyState = Template.bind({});
 EmptyState.args = {
   data: [],
 };
+
+const BreakpointTemplate: ComponentStory<typeof DataList> = args => {
+  const {
+    data,
+    /* See useCollectionQuery for example on how to load more */
+    // refresh,
+    // nextPage,
+    // loadingRefresh,
+    // loadingNextPage,
+    loadingInitialContent,
+  } = useCollectionQuery<ListQueryType>({
+    query: LIST_QUERY,
+    queryOptions: {
+      fetchPolicy: "network-only",
+      nextFetchPolicy: "cache-first",
+      client: apolloClient,
+    },
+    getCollectionByPath(items) {
+      return items?.allPeople;
+    },
+  });
+
+  /* For when we start creating a loading state */
+  // const { loading } = getLoadingState(
+  //   loadingInitialContent,
+  //   loadingRefresh,
+  //   loadingNextPage,
+  // );
+
+  const items = data?.allPeople.edges || [];
+  const mappedData = items.map(({ node }) => ({
+    label: node.name,
+    home: node.homeworld.name,
+    tags: [
+      node.gender,
+      node.hairColor?.split(", "),
+      node.skinColor?.split(", "),
+    ],
+    homePopulation: node.homeworld.population?.toLocaleString(),
+    created: new Date(node.created),
+  }));
+
+  return (
+    <DataList
+      {...args}
+      data={(args.data as typeof mappedData) || mappedData}
+      headers={{
+        label: "Name",
+        home: "Home world",
+        tags: "Attributes",
+        homePopulation: "Home world population",
+        created: "Created",
+      }}
+      loading={loadingInitialContent}
+    >
+      <DataList.Layout size="lg">
+        {(item: DataListItemType<typeof mappedData>) => (
+          <Grid alignItems="center">
+            <Grid.Cell size={{ xs: 5 }}>
+              <Grid alignItems="center">
+                <Grid.Cell size={{ xs: 6 }}>{item.label}</Grid.Cell>
+                <Grid.Cell size={{ xs: 6 }}>{item.home}</Grid.Cell>
+              </Grid>
+            </Grid.Cell>
+            <Grid.Cell size={{ xs: 2 }}>{item.homePopulation}</Grid.Cell>
+            <Grid.Cell size={{ xs: 3 }}>{item.tags}</Grid.Cell>
+            <Grid.Cell size={{ xs: 2 }}>{item.created}</Grid.Cell>
+          </Grid>
+        )}
+      </DataList.Layout>
+      <DataList.Layout size={"xs"} showHeader={true}>
```

Comment:
I wonder if `showHeader` is better as `withHeader` english and semantically speaking.

Also, is supposed to be `false`?

Reply:
It should be `false`. I feel `showHeader` makes more sense but I am not super attached to it